src/toast/tests/ops_stokes_weights.py

```diff
@@ -48,15 +48,15 @@ def create_data(self, I_sky, Q_sky, U_sky, hwp):
             create_dist="pixel_dist",
             detector_pointing=detpointing,
         )
-        pixels.apply(data)
         weights = ops.StokesWeights(
             mode="IQU",
             hwp_angle=hwp_name,
             detector_pointing=detpointing,
             fp_gamma="gamma",
             IAU=False,
         )
-        weights.apply(data)
+        pipe = ops.Pipeline(operators=[pixels, weights])
```

Comment:
Thanks for this change, which will now test the stokes weights kernel if GPU is enabled.